src/components/DpDataTable/DpResizableColumn.vue

```diff
@@ -100,6 +118,10 @@ export default {
       document.querySelector('body').removeEventListener('mousemove', this.namedFunc)
       document.querySelector('body').removeEventListener('mouseup', this.stopResize)
       document.querySelector('body').classList.remove('resizing')
+    },
+
+    updateSessionStorage (field, width) {
+      sessionStorage.setItem(`data-col-field=${field}`, width)
```

Comment:
I'd suggest to prefix the key with `dpDataTable:` for better readability when looking through a bunch of storage items in the developer tools.

Reply:
Do you mean: `data-col-field=dpDataTable:${field}` ?

Reply:
no, i thought of `dpDataTable:data-col-field=${field}` but i think we do not have any sufficient rules for naming sessionStorage entries at all, we can always discuss this later...

however i find it a little confusing that there the `updateSessionStorage` method is defined in two places but with slightly differing implementations.